tests/DocumentationExamplesTest.php

```diff
@@ -1180,6 +1184,223 @@ public function testIndex_example_2()
         $this->assertEquals('cuisine_1_name_1', $indexName);
     }
 
+    // Start Transactions Intro Example 1
+    private function updateEmployeeInfo1(Manager $manager, Session $session)
+    {
+        $employees = (new Database($manager, "hr"))->employees;
+        $events = (new Database($manager, "reporting"))->events;
```

Comment:
I'm also not sure how I feel about revealing the Manager here. If we can avoid it, I'd rather not suggest that users work with Managers directly nor suggest that they manually construct Database and Collection.

Can you inject the Database object (or a Client) instead? From there, you can use `$db->employees` and `$db->events`, respectively.

Reply:
Done, using client